tests/common/server.rs

```diff
@@ -213,30 +219,46 @@ pub async fn start_server(
 		extra_args.push_str(format!(" --tick-interval {sec}s").as_str());
 	}
 
-	let start_args = format!("start --bind {addr} memory --no-banner --log trace --user {USER} --pass {PASS} {extra_args}");
-
-	info!("starting server with args: {start_args}");
+	'retry: for _ in 0..3 {
+		let port: u16 = rng.gen_range(13000..14000);
```

Comment:
In addition to closing the servers, and retrying 3 times, is it also worth just increasing the port range so that there is less chance of port conflicts?

Reply:
That could help, I tried it when I was figuring out what was causing the errors and I still got failures. I wasn't sure about the rational behind the 1000 port range so I kept it in.